client/android/app/build.gradle

```diff
@@ -46,7 +46,7 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        minSdkVersion 21
+        minSdkVersion 23
```

Comment:
Does it work with 21?

Reply:
Geez, 2015, almost 10 years ago: https://developer.android.com/tools/releases/platforms#6.0
I guess switching the rest of Flet to 23 is fine. Will keep an eye on it.

Reply:
if it is from 2015, that means it is from android 6 right?

i think this year with android 15, they are making that the minimum supported by android.

Reply:
found where they told that in the docs : https://android-developers.googleblog.com/2024/05/the-second-beta-of-android-15.html